indico/modules/events/papers/templates/emails/judgment_to_author.html

```diff
@@ -30,7 +30,7 @@
     {% if paper.judgment_comment %}
         <dl>
             <dt style="width: 30%; float: left;">{% trans %}Comment:{% endtrans %}</dt>
-            <dd style="width: 70%; float: left; margin: 0; min-height: 20px; white-space: pre-wrap; word-break: break-all; padding-left: 20px; box-sizing: border-box;">
```

Comment:
These templates are for HTML emails (ie NO external CSS whatsoever) so I think removing box-sizing from any email template (`...../templates/emails/...`) may be risky - I doubt email clients default to border-box...

Reply:
Hm. Ok, let's revert those then.